codespell_lib/_codespell.py

```diff
@@ -389,11 +444,14 @@ def parse_options(args):
     parser.add_argument('-C', '--context', type=int, metavar='LINES',
                         help='print LINES of surrounding context')
     parser.add_argument('--config', type=str,
-                        help='path to config file.')
+                        help='path to config file.',
+                        ).complete = shtab.FILE
     parser.add_argument('--toml', type=str,
-                        help='path to a pyproject.toml file.')
+                        help='path to a pyproject.toml file.',
+                        ).complete = TOML_FILE
     parser.add_argument('files', nargs='*',
-                        help='files or directories to check')
+                        help='files or directories to check',
+                        ).complete = shtab.FILE
```

Comment:
Does the file one also suggest directories on their own? E.g. given:
foo\bar which contains a single file baz.txt, can I tab complete to just foo\bar?

Reply:
FILE complete not only normal files but also directories